lib/plugins/scoreboard.js

```diff
@@ -57,6 +58,47 @@ function inject (bot) {
     }
   })
 
+  bot.on('teams', (packet) => {
+    if (packet.mode === 0) {
+      const team = {}
+      team.displayName = packet.name
+      team.prefix = packet.prefix
+      team.suffix = packet.suffix
+      team.friendlyFire = packet.friendlyFire
+      team.nameTagVisibility = packet.nameTagVisibility
+      team.collisionRule = packet.collisionRule
+      team.color = packet.color
+      team.players = packet.players
+      teams[packet.team] = team
+    }
+
+    if (packet.mode === 1) {
+      delete teams[packet.team]
+    }
+
+    if (packet.mode === 2) {
+      if (teams[packet.team] === undefined) return
```

Comment:
Shouldn't we create a team then?

Reply:
the vanilla client just assumes the team exist, its possible this line isnt even needed.

Reply:
Have you actually tested this?

Reply:
yes ive run it on my own bot on several non vanilla servers already, seems to work fine